doc/tutorials/introduction/windows_install/windows_install.markdown

```diff
@@ -24,6 +24,84 @@ Installation by Using the Pre-built Libraries {#tutorial_windows_install_prebuil
 
 -#  To finalize the installation go to the @ref tutorial_windows_install_path section.
 
+Installation by Using git-bash (version>=.14.1) and cmake (version >=3.9.1)
+===============================================================
+
+-#  You must download [cmake (version >=3.9.1)](https://cmake.org) and install it. You must add cmake to PATH variable during installation
+
+-#  You must install [git-bash (version>=.14.1)](https://git-for-windows.github.io/). Don't add git to PATH variable during installation
+
+-#  Run git-bash. You observe a command line window.
+Suppose you want to build opencv and opencv_contrib in c:/lib
+
+-#  In git command line enter following command (if folder does not exist) :
+@code{.bash}
+mkdir /c/lib
+cd /c/lib
+@endcode
+
+-#  save this script with name installOCV.sh in c:/lib
+
+        @code{.bash}
+myRepo=$(pwd)
```

Comment:
Add script header, something like:
```
#!/bin/bash -e
```

To stop script execution on errors.

Reply:
OK